time-clock/.gitignore

```diff
@@ -5,7 +5,7 @@
 #   git config --global core.excludesfile '~/.gitignore_global'
 
 #Ignore database config
-config/database.yml
+./config/database.yml
```

Comment:
Is this actually working?
I mean, I just made some changes to this file, and adding the dot won't make anything I think